resources/authgear/templates/en/web/authflowv2/__divider.html

```diff
@@ -0,0 +1,15 @@
+{{ define "authflowv2/__divider.html" }}
+
+<div class="divider__container {{$.Classname}}">
+    {{/* TODO(authflowv2): Template don't support passing dynamic variable so
+    hardcoding "or" here. Need to implement a function like helm include
+    function
+    https://helm.sh/docs/howto/charts_tips_and_tricks/#know-your-template-functions
+    */}}
+  <hr class="divider"></hr>
+  <p class="divider_text">{{template "or-label"}}</p>
```

Comment:
I think we should use new translation strings for v2 ui.
Lets prefix the translation ids with `v2-`. So we should add a `v2-or-label`.
@louischan-oursky See if any thoughts?

Reply:
OK. Let's use `v2-` prefix.